client/menus/main.lua

```diff
@@ -138,7 +145,8 @@ menu.onClose = function()
 end
 
 lib.callback.register('customs:client:vehicleProps', function()
-    return QBCore.Functions.GetVehicleProperties(vehicle)
+    return lib.getVehicleProperties(vehicle)

```

Comment:
qb uses [`QBCore.Functions.SetVehicleProperties`](https://docs.qbcore.org/qbcore-documentation/qb-core/client-function-reference#qbcore.functions.setvehicleproperties)  to set the vehicle properties back, will this not break that?